lib/src/client/options.dart

```diff
@@ -18,8 +18,25 @@ import 'dart:io';
 
 import 'dart:math';
 
+import 'package:http2/transport.dart';
+
 import '../shared/security.dart';
 
+class Http2Streams {
+  final Stream<List<int>> incoming;
+  final StreamSink<List<int>> outgoing;
+  final Future<void> done;
```

Comment:
`done` is like `incoming.done`. Docs from `Socket.done`: `Get a future that will complete when the consumer closes...`. What i understand is that the consumer is `incoming`. But i am a newcomer to dart.

Reply:
A `Stream` has no `.done`. Only a `StreamSink`.
The `outgoing` StreamSink is where you send messages you send to the consumer, and it will complete with `.done` when it closes its `StreamSubscription`.